neo4j-cli/aura/internal/subcommands/import/job/cancel.go

```diff
@@ -0,0 +1,56 @@
+package job
+
+import (
+	"fmt"
+	"github.com/neo4j/cli/common/clicfg"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/api"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/output"
+	"github.com/spf13/cobra"
+	"log"
+	"net/http"
+)
+
+func NewCancelCommand(cfg *clicfg.Config) *cobra.Command {
+	var (
+		projectId string
+		jobId     string
+	)
+
+	const (
+		projectIdFlag = "project-id"
+	)
+
+	cmd := &cobra.Command{
+		Use:   "cancel <id>",
+		Short: "Cancel a job by id",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("jobId is required")
+			}
```

Comment:
The option `Args:  cobra.ExactArgs(1),` will do that for you. Check `instance/get.go` as an example. This way we can avoid the `PreRunE` completely